openapi3filter/validate_request.go

```diff
@@ -61,6 +61,14 @@ func ValidateRequest(c context.Context, input *RequestValidationInput) error {
 
 	// Security
 	security := operation.Security
+	// If there aren't any security requirements for the operation
+	if security == nil {
+		if input.Route.Swagger == nil {
```

Comment:
Is this case really a possibility by then?

Reply:
I am not sure about which case you're talking about since there are two if statements in this block. However, I believe that both are possible since neither of them have been checked. We don't have to look far to see whether the check has been made since this function is exported, we can just have to look whether the check is made inside of this function and it isn't.